### ISSUE TITLE
[Other] No VA.gov Experience Standard for the issue found. (00.00.5)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#54707]
#### VFS team name
GovCIO-VFEP

#### VFS product name
GI Bill Comparison Tool

#### VFS feature name
Institution Ratings System

#### Point of Contact/Reviewers
Brian DeConinck - @briandeconinck - Accessibility

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
No VA.gov Experience Standard for the issue found.

### Issue Details
When comparing two institutions, the table cells for each institution's overall rating have a very verbose `aria-label` that is not necessary in this context.

### Link, screenshot or steps to recreate
Overall rating cells are coded as:
```
<div aria-label="Overall Rating for UNIVERSITY OF MARYLAND-COLLEGE PARK is 3.1 out of 4.">3.1 out of a possible 4 stars</div>
```
or:
```
<div aria-label="Overall rating for UNIVERSITY OF MARYLAND COLLEGE PARK-WASHINGTON DC has not yet been rated by veterans">Not yet rated by Veterans</div>
```

Note the `aria-label` text which duplicates the visible text in the cell.

### VA.gov Experience Standard
[Category Number 00, Issue Number 00](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Platform Recommendation
These comparison tables are a little over-engineered in terms of ARIA (I'm not totally clear why they couldn't just be `<table>` elements, but that's neither here nor there). But they do have a clear table structure that's communicated to screen reader users. That means a screen reader user should be able to navigate through each table cell and hear all of the necessary context just from the on-screen text for these rating cells. There's no need to include the `aria-label`.

---

### VFS Guidance

- Close the ticket when the issue has been resolved or validated by your Product Owner
- If your team has additional questions or needs Platform help validating the issue, please comment on the ticket
- Some feedback provided may be out of scope for your iteration of the product, however, Platform's OCTO leadership has stated that all identified issues need to be documented and it is still your responsibility to resolve the issue.
- If you do not believe that this Staging Review issue ticket is the responsibility of your team, comment below providing an explanation and who you believe is responsible. Please tag the Point of Contact/Reviewers. Governance team will research and will follow up.
